content/blog/automation-api-dotnet/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Automate Your Infrastructure with Automation API and C#"
+date: tbd
+meta_desc: "tbd"
+meta_image: automation_api.png
+authors:
+- joshua-studt
```

Comment:
FYI: you'll want to add a `.toml` file to `data/team/team` for this to render correctly.